### ISSUE TITLE
cy.check and cy.uncheck log in Command Log irregardless of actual elements checked / unchecked

### ISSUE THREAD

**Issue by jennifer-mann** [cypress-io/cypress#94]
**Description**
`cy.check()` and `cy.uncheck()` log in the Command Log for each element found in it's current subject. The commands don't take into account the actual number of elements that are checked or unchecked. 

They also do not check the values passed into the command to see how many elements match the command and to only log those matching.

**Code**
```javascript
cy.get("input[type='checkbox']").uncheck() // this actually only unchecked 2 checkboxes
cy.get("input[type='checkbox']").check(["2"])  // this actually only checked 1 checkbox
```

**Additional Info**
This command logged for every checkbox found
![screen shot 2016-01-21 at 3 50 36 pm](https://cloud.githubusercontent.com/assets/1271364/12494294/59dca3d6-c057-11e5-975b-8a19a616969b.png)

**Comment by jennifer-mann** [comment]
This is fixed in the upcoming `0.14.0` release.

**Comment by brian-mann** [comment]
Fixed in [`0.14.0`](https://github.com/cypress-io/cypress/wiki/Changelog#0140-03082016).
